### PR TITLE
[FEAT] Faction Point Cutover

### PR DESCRIPTION
**Pull request by c-py**
# Description

Disables Faction Points when Kingdom goes live ( 14th June). Introduces new ERC 1155 resource `Faction Mark`. Different features behave differently:

**Chores**
Faction points are removed from chores

**Deliveries**
Faction points are removed from deliveries

**Chicken Rescue**
Starts putting out Marks

**Donations**
Feature is disabled

## How to Test

All features are toggled by a single switch `FACTION_POINT_CUTOFF`. In order to test use `FACTION_POINT_CUTOFF` in the following scenarios:

1. `FACTION_POINT_CUTOFF` in future.

2. `FACTION_POINT_CUTOFF` in past.

Visit each of the four features listed above to make sure they indeed do not give out faction points.

## Type of change

Please delete options that are not relevant.

- [X] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

localhost

# Checklist:

- [X] Title of the PR is relevant and is prefixed with [FEAT], [CHORE] or [FIX]
- [ ] Screenshot if it includes any UI changes
- [ ] I have read the contributing guidelines and agree to the T&Cs
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/features/world/ui/portals/ChickenRescue.tsx [hunk 1]
```diff
@@ -10,6 +10,7 @@ import { Label } from "components/ui/Label";
 import coins from "assets/icons/coins.webp";
 import factions from "assets/icons/factions.webp";
 import flagIcon from "assets/icons/faction_point.webp";
+import markIcon from "assets/icons/faction_mark.webp";
```
Comment (c-py) [inline src/features/world/ui/portals/ChickenRescue.tsx:13]: Temporary hack art - pending better art

temporary

**Review by spencerdezartsmith — APPROVED** [review]
🤌

**Comment by eliasSFL** [comment]
Love that we are taking a more automatic approach to these releases!

File: src/features/game/events/landExpansion/donateToFaction.ts [hunk 2]
```diff
@@ -5,6 +5,8 @@ import { getDayOfYear } from "lib/utils/time";
 
 const DAILY_SFL_DONATION_LIMIT = 500;
 
+export const FACTION_POINT_CUTOFF = new Date("2024-06-14T00:00:00Z");
```
Comment (brian-man) [inline src/features/game/events/landExpansion/donateToFaction.ts:8]: You should be able to abbreviate:
`2024-06-14Z`

File: src/features/game/events/minigames/claimMinigamePrize.test.ts [hunk 3]
```diff
@@ -1,7 +1,18 @@
 import { TEST_FARM } from "features/game/lib/constants";
 import { claimMinigamePrize } from "./claimMinigamePrize";
+import { FACTION_POINT_CUTOFF } from "../landExpansion/donateToFaction";
+import Decimal from "decimal.js-light";
 
 describe("minigame.prizeClaimed", () => {
+  beforeEach(() => {
+    jest.useFakeTimers();
+    jest.setSystemTime(new Date("2024-05-01"));
```
Comment (brian-man) [inline src/features/game/events/minigames/claimMinigamePrize.test.ts:9]: Add the `Z`.

File: src/features/game/events/minigames/claimMinigamePrize.test.ts [hunk 4]
```diff
@@ -215,4 +226,121 @@ describe("minigame.prizeClaimed", () => {
       ].prizeClaimedAt
     ).toEqual(date.getTime());
   });
+
+  it("claims a faction marks prize", () => {
+    const date = new Date("2024-05-05T00:00:00");
```
Comment (brian-man) [inline src/features/game/events/minigames/claimMinigamePrize.test.ts:231]: Might want to make sure all timestamps in all the affected files have the trailing `Z`.

File: src/features/game/lib/processEvent.ts [hunk 5]
```diff
@@ -125,10 +125,11 @@ export const maxItems: Inventory = {
   "Love Letter": new Decimal(400),
 
   // Emblems
-  "Goblin Emblem": new Decimal(500),
-  "Bumpkin Emblem": new Decimal(500),
-  "Sunflorian Emblem": new Decimal(500),
-  "Nightshade Emblem": new Decimal(500),
+  "Goblin Emblem": new Decimal(90_000),
```
Comment (brian-man) [inline src/features/game/lib/processEvent.ts:128]: If someone has > 90000 to claim (on Jun 14) what happens? Do they claim in batches or just never claim b/c it's over the limit?

**Pull request merged**